templates/admin/status.php

```diff
@@ -0,0 +1,29 @@
+<?php
+// Exit if accessed directly.
+defined( 'ABSPATH' ) || exit;
+?>
+<div>
```

Comment:
Please fix the formatting (e.g. tabs for nested elements), also add the same wrapper and page title with divider like we use for other pages, lines 5-7 here https://github.com/hivepress/hivepress/blob/master/templates/admin/themes.php

Also check if WordPress has similar tables somewhere, maybe in Settings/General, you can use the same table class to avoid custom CSS